packages/office-ui-fabric-react/src/components/ComponentStatus/ComponentStatusState.tsx

```diff
@@ -0,0 +1,371 @@
+import { IComponentStatusProps, TestCoverageStatus } from '@uifabric/example-app-base';
+
+export interface IComponentStatusState {
+  [key: string]: IComponentStatusProps;
+}
+
+export const ComponentStatusState: IComponentStatusState = {
```

Comment:
shouldn't we track that in the individual component pages?

Reply:
Will it be included in the bundles we generate?

Reply:
@cschleiden I agree that it is better to track them in the individual component pages. Nonetheless, because as of this moment, ComponentStatusPage renders all component status from the componentstatusstate dictionary, I will create a bug to address this issue in a separate PR.

This will not be included in the bundle oufr generates because in the next commits I made, the componentstatus code was moved to the demo dir, so it will not be shipped.